src/_includes/services_page_intro.liquid

```diff
@@ -1,5 +1,40 @@
 <header>
-  <div class="services-page-intro__image"></div>
+  <div class="services-page-intro__image">
```

Comment:
This class looks to now be the block element. It's classname could be `services-page-intro`. It's elements within would then need to follow suit with the name change.